.github/workflows/wheezy_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Wheezy CI
+on:
+  pull_request:
+    paths:
+      - "Formula/patchelf.rb"
+      - "Formula/binutils.rb"
+jobs:
+  wheezy_tests:
+    if: github.repository == 'Homebrew/homebrew-core' && github.event_name == 'pull_request' && ! contains(github.event.pull_request.labels.*.name, 'CI-syntax-only')
+    runs-on: ubuntu-latest
+    env:
+      HOMEBREW_CORE_GIT_REMOTE: ${{github.event.repository.html_url}}
+      HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Hack curl version
+        run: |
+          sed -i 's/HOMEBREW_MINIMUM_CURL_VERSION="7.41.0"/HOMEBREW_MINIMUM_CURL_VERSION="7.25.0"/g' /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/brew.sh
+      - name: Setup bottles folder
+        run: |
+          rm -rf ~/bottles
+          mkdir ~/bottles
+      - name: Run Docker
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+          HOMEBREW_NO_ANALYTICS: 1
+          HOMEBREW_DEVELOPER: 1
+          HOMEBREW_CURL_PATH: "/usr/bin/curl"
+          HOMEBREW_GIT_PATH: "/usr/bin/git"
+        run: |
+          set -eux
+          docker pull homebrew/debian7:latest
+          docker run -v /home/linuxbrew:/home/linuxbrew -v ~/bottles:/home/linuxbrew/bottles -u root --env-file <(env | grep 'HOMEBREW\|GITHUB') homebrew/debian7:latest bash -c 'cd /root && brew test-bot --only-formulae && cp *.bottle.* /home/linuxbrew/bottles'
```

Comment:
Is it possible to use the GitHub Actions official Docker support and run in that image directly rather than doing this?

Reply:
The image is too old to run GitHub Actions stuff directly in the container, so we have to use another layer of Docker-ness to get this to build.

Reply:
To clarify, it's not that the image is too old, but the software on the image is too old. We currently build a newer version of Git when creating the image and that's it. To get GitHub Actions to run on it, we need to update a whole lot more software.

Reply:
Yes, it's exactly the same question as @dawidd6 's above.
> Why can't we just run the whole workflow in container?

It's just too old. I think some certs needs to be updated, openssl too probably. Maybe curl needs to be rebuilt from source.

The "normal" linux workflow for all the other bottles will be much easier, this one is an "exception" for patchelf and binutils only, and is a little bit more ... exotic. At least we have a workflow for it now :)

Reply:
Ok, fine with me 👍🏻